egs/audioset/AT/local/generate_audioset_manifest.py

```diff
@@ -0,0 +1,134 @@
+import argparse
+import csv
+import glob
+import logging
+
+import torch
+from lhotse import CutSet, Fbank, FbankConfig, LilcomChunkyWriter
+from lhotse.audio import Recording
+from lhotse.cut import MonoCut
+from lhotse.supervision import SupervisionSegment
+
+from icefall.utils import get_executor
+
+torch.set_num_threads(1)
+torch.set_num_interop_threads(1)
+
```

Comment:
Could you follow
https://github.com/k2-fsa/icefall/blob/42de4591107868816d240c991b1e8acdfdfd090b/egs/librispeech/ASR/local/compute_fbank_librispeech.py#L19-L24
to give a short description about what this file does?

Reply:
Added